src/Models/Services/Utils.ts

```diff
@@ -36,6 +36,7 @@ import {
     IConsoleLogFunction,
     TimeSeriesData
 } from '../Constants/Types';
+import * as d3 from 'd3';
```

Comment:
hmm, instead of importing whole d3, can we try to only import the format method from that package? of I noticed that you can import it directly from submodule like:
```suggestion
import { format } from 'd3-format';
```

can you please try this and confirm it still works?